docs/MigrationGuide.md

```diff
@@ -15,10 +15,10 @@ To see the biggest differences please consult the [changelog](Changelog.md).
 
 * [#2983](https://github.com/TouK/nussknacker/pull/2983) Extract Permission to extensions-api
   * Moved `pl.touk.nussknacker.ui.security.api.Permission` (security module) to `pl.touk.nussknacker.security.Permission` (extension-api module)
-* [#3029](https://github.com/TouK/nussknacker/pull/3029) `KafkaConfig` has new field `schemaRegistryCacheConfig: SchemaRegistryCacheConfig`.
+* [#3029](https://github.com/TouK/nussknacker/pull/3029) `KafkaConfig` has new field `schemaRegistryCacheConfig: SchemaRegistryCacheConfig`. Flink state compatibility has been broken!
```

Comment:
I think, that we've already talk on some meeting, that if change can break someone's runtime, we should have a separate section for that, because such notice may be not visible. WDYT?